### ISSUE TITLE
Can't compile the plugin on windows

### ISSUE THREAD

**Issue by rwang5203** [basler/gst-plugin-pylon#66]
The bug:
Library nvbufsurface found: NO
Run-time dependency CUDA (modules: cudart) found: NO (tried system)
Message: Deepstream or CUDA not found, skipping NVMM support
Run-time dependency valgrind found: NO (tried pkgconfig)
Run-time dependency gstreamer-plugins-base-1.0 found: YES 1.22.6
Dependency gstpylon found: YES 0.7.0 (overridden)
Program C:/gstreamer/1.0/msvc_x86_64/libexec/gstreamer-1.0/gst-plugins-doc-cache-generator found: NO
docs\meson.build:32: WARNING: GStreamer plugin inspector for documentation not found, can't update the cache
Program hotdoc found: NO
Message: Hotdoc not found, not building the documentation
Message: Meta python bindings disabled
Configuring config.h using configuration
Build targets in project: 8

To Reproduce:
meson setup build --prefix=%GSTREAMER_1_0_ROOT_MSVC_X86_64%

Expected behavior
The paths are all correct, not sure what path is wrong.

**Comment by rwang5203** [comment]
That was fixed by downloading the development installer. Now a new compilation error appeared.

Library nvbufsurface found: NO
Run-time dependency CUDA (modules: cudart) found: NO (tried system)
Message: Deepstream or CUDA not found, skipping NVMM support
Run-time dependency valgrind found: NO (tried pkgconfig)
Run-time dependency gstreamer-plugins-base-1.0 found: YES 1.22.6
Dependency gstpylon found: YES 0.7.0 (overridden)
Program C:/gstreamer/1.0/msvc_x86_64/libexec/gstreamer-1.0/gst-plugins-doc-cache-generator found: NO
docs\meson.build:32: WARNING: GStreamer plugin inspector for documentation not found, can't update the cache
Program hotdoc found: NO
Message: Hotdoc not found, not building the documentation
Message: Meta python bindings disabled
Configuring config.h using configuration
Build targets in project: 8

**Comment by rwang5203** [comment]
Library nvbufsurface found: NO
Run-time dependency CUDA (modules: cudart) found: NO (tried system)
Message: Deepstream or CUDA not found, skipping NVMM support
Run-time dependency valgrind found: NO (tried pkgconfig)
Run-time dependency gstreamer-plugins-base-1.0 found: YES 1.22.6
Dependency gstpylon found: YES 0.7.0 (overridden)
Program C:/gstreamer/1.0/msvc_x86_64/libexec/gstreamer-1.0/gst-plugins-doc-cache-generator found: NO
docs\meson.build:32: WARNING: GStreamer plugin inspector for documentation not found, can't update the cache
Program hotdoc found: NO
Message: Hotdoc not found, not building the documentation
Message: Meta python bindings disabled
Configuring config.h using configuration
Build targets in project: 8

**Comment by thiesmoeller** [comment]
What is the error, that you get?
Is meson failing or ninja ?

**Comment by rwang5203** [comment]
Here is the error message it produces when I try `meson setup build --prefix=%GSTREAMER_1_0_ROOT_MSVC_X86_64%`

```
Program C:/gstreamer/1.0/msvc_x86_64/libexec/gstreamer-1.0/gst-plugins-doc-cache-generator found: NO
docs\meson.build:32: WARNING: GStreamer plugin inspector for documentation not found, can't update the cache
Program hotdoc found: NO
Message: Hotdoc not found, not building the documentation
Message: Meta python bindings disabled
Configuring config.h using configuration
Build targets in project: 8
```

And when I ignore and run `ninja -C build`, it runs into error again. I think that the meson failed the compilation.

Environment: Windows 11, CUDA 12.2, Pylon 7.4.0, gstreamer 1.22.6.

**Comment by rwang5203** [comment]
This issue is resolved after reinstalling pylon 7.3.0. Thanks!
